state/runtime/evm/instructions.go

```diff
@@ -486,10 +409,10 @@ func opSload(c *state) {
 	var gas uint64
 
 	if c.config.Berlin {
-		if _, slotPresent := c.host.ContainsAccessListSlot(c.msg.Address, bigToHash(loc)); !slotPresent {
+		if _, slotPresent := c.host.ContainsAccessListSlot(c.msg.Address, uint256ToHash(loc)); !slotPresent {
```

Comment:
Minor optimization: capture the result of the `uint256ToHash` into a variable and reuse it.